gnomad_qc/v3/sample_qc/subpop_analysis.py

```diff
@@ -356,10 +357,16 @@ def main(args):  # noqa: D103
         default=0.90,
     )
     parser.add_argument(
-        "--max-mislabeled-training-samples",
-        help="Maximum number of training samples that can be mislabelled",
+        "--max-number-mislabeled-training-samples",
```

Comment:
Maybe change to use: https://docs.python.org/3/library/argparse.html#mutual-exclusion